test/distributed/test_inductor_collectives.py

```diff
@@ -1676,21 +1683,49 @@ def _reorder_communication_preserving_peak_memory(
                 "reorder_for_compute_comm_overlap": True,
                 "reorder_for_compute_comm_overlap_passes": [
                     _reorder_communication_preserving_peak_memory,
-                    sink_waits_iterative,
                 ],
             }
         ):
             compiled = torch.compile(func)
             code = run_and_get_triton_code(compiled, *inputs, **self.get_world_trs())
         # NOTE: The first return value should be the output of the first wait_tensor.
         # We want to make sure no unneccessary copy is made.
-        (FileCheck().check("all_gather_into_tensor_out").run(code))
+        (
+            FileCheck()
+            .check_count(
```

Comment:
just out of curiosity, any reason you used 3 separate FileCheck instances instead of just doing it all in one?

Reply:
Oh, thanks, it can be done as a one check :)